docs/man/qdstat.8.adoc

```diff
@@ -412,6 +412,285 @@ The operational status of this auto link:
 lastErr::
 The description of the last attach failure that occurred on this auto link.
 
+qdstat --policy
+~~~~~~~~~~~~~~~
+
+Maximum Concurrent Connections::
+The maximum number of concurrent client connections allowed for this router.
+
+Maximum Message Size::
+The maximum size in bytes of AMQP message transfers allowed for this router as messages enter the
```

Comment:
```suggestion
The maximum size in bytes of AMQP message transfers allowed for this router.
```
I think the next sentence addresses the 'entering the network' better than the fragment.